tests/scripts/all.sh

```diff
@@ -1965,6 +1965,7 @@ component_test_psa_crypto_config_accel_ecdsa () {
     scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_ECB_NO_PADDING
 
     # SHA384 needed for some ECDSA signature tests.
+    scripts/config.py -f tests/include/test/drivers/config_test_driver.h set MBEDTLS_SHA224_C
```

Comment:
Is this needed? What was failing?

Reply:
According to the `all.sh` script it seems that the pattern is similar to the previous comment, so I'll wait for that one to be solved/clarified before looking into this one

Reply:
I think that's correct, however please adapt the comment above, for example "These hashes are needed", rather than SHA384.